osync.sh

```diff
@@ -2072,7 +2078,9 @@ function InitRemoteOSDependingSettings {
 		fi
 	fi
 	if [ "$RSYNC_COMPRESS" == true ]; then
-		if [ "$LOCAL_OS" != "MacOSX" ] && [ "$REMOTE_OS" != "MacOSX" ]; then
+		if [ "$LOCAL_OS" == "Qnap" ] || [ "$REMOTE_OS" == "Qnap" ]; then
```

Comment:
This was added since -zz is not supported on Qnap... :( 

          --copy-dest=DIR         ... and include copies of unchanged files
          --link-dest=DIR         hardlink to files in DIR when unchanged
      -z, --compress              compress file data during the transfer
          --compress-level=NUM    explicitly set compression level
          --skip-compress=LIST    skip compressing files with a suffix in LIST
      -C, --cvs-exclude           auto-ignore files the same way CVS does